py/kubeflow/kfctl/testing/pytests/endpoint_ready_test.py

```diff
@@ -15,7 +15,7 @@
 
 # There's really no good reason to run test_endpoint during presubmits.
 # We shouldn't need it to feel confident that kfctl is working.
-@pytest.mark.skipif(os.getenv("JOB_TYPE") == "presubmit",
+@pytest.mark.skipif(os.getenv("JOB_TYPE") == "periodic",
```

Comment:
Why don't you want to skip the endpoint test in the presubmits?

Reply:
this is a temp change for this function.  will rebase and drop it after I can make sure the function works.

Reply:
Got it thanks.